ydb/core/base/id_wrapper.h

```diff
@@ -0,0 +1,80 @@
+#pragma once
```

Comment:
add some test (using all methods and operators) to ensure successful compilation at least